Source/WebCore/dom/Element.h

```diff
@@ -100,6 +100,12 @@ struct ShadowRootInit;
 
 using ExplicitlySetAttrElementsMap = HashMap<QualifiedName, Vector<WeakPtr<Element, WeakPtrImplWithEventTargetData>>>;
 
+struct ContentVisibilityData {
+    WTF_MAKE_STRUCT_FAST_ALLOCATED;
+    bool isSkippedContent { false };
+    ContentVisibility contentVisibility;
+};
```

Comment:
If there is nothing else to put here, this doesn't really need a heap allocated struct. Those fields could just sit directly in the rare data and take less space.

Reply:
Hmm, we will have more flags here for `content-visibility: auto` to indicate if the element is [relevant to the user](https://w3c.github.io/csswg-drafts/css-contain-2/#relevant-to-the-user), something like:
```
    bool isIntersectingViewport { false };
    bool subtreeHasFocus { false };
    bool subtreeHasSelection { false };
```
See the WIP-patch in https://bugs.webkit.org/show_bug.cgi?id=236711, looks like there isn't complex function there.
I'm fine to put `isSkippedContent` and `contentVisibility` in `ElementRareData`.

Reply:
Hi Cathie, I think the idea was to get rid of the ContentVisibilityData completely, or at least not dynamically allocate it. Anyway I can fix this tomorrow morning.

Reply:
I pushed a new version for what I think was the initial suggestion.

Reply:
The failing win test has been flaky for a while so I think we can ignore it:
https://ews-build.webkit.org/#/builders/10

So this is ready for review again.